underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -107,6 +102,24 @@ public static HdfsUnderFileSystem createInstance(
   public HdfsUnderFileSystem(AlluxioURI ufsUri, UnderFileSystemConfiguration conf,
       Configuration hdfsConf) {
     super(ufsUri, conf);
+
+    // Create the supported HdfsAclProvider if possible.
+    HdfsAclProvider hdfsAclProvider = new NoopHdfsAclProvider();
+    try {
+      Object o = Class.forName("alluxio.underfs.hdfs.acl.SupportedHdfsAclProvider").newInstance();
```

Comment:
javadoc that this class may not be available as .class file, so we use string literal?

Reply:
Done.